src/partition/repart.c

```diff
@@ -1316,6 +1320,41 @@ static int config_parse_size4096(
         return 0;
 }
 
+static int config_parse_block_size(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        uint64_t *blksz = data;
+        int r;
+
+        assert(rvalue);
+        assert(data);
+
+        r = safe_atou64(rvalue, blksz);
```

Comment:
please use parse_size()

Reply:
Are you sure we should use parse_size() here? I don't think its very useful to allow size suffixes in this case, we don't allow those for sector sizes either.

Reply:
we should be systematic here. i mean, the example this adds to the docs even says "4K" block size, so i think there's a lot of value to just accept this like we accept it everywhere